File: src/main/java/com/web/fitmaster/model/Package.java
package com.web.fitmaster.model;

import com.web.fitmaster.model.enums.PackageStatus;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "packages")
public class Package {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long packageId;

    private String name;

    private int durationInDays;

    private double price;

    private String description;

    @Enumerated(EnumType.STRING)
    private PackageStatus status;



}

File: frontend/src/types/package.ts
export interface Package {
  id: string;
  name: string;
  price: number;
  durationInDays: number;
  status:string;
  description:string;
}

export interface CreatePackageRequest {
  name: string;
  price: number;
  durationInDays: number;
  description: string;
  status:string;
}

export interface UpdatePackageRequest {
  name?: string;
  price?: number;
  durationInDays?: number;
  description?: string;
}
